### ISSUE TITLE
Python3 support 

### ISSUE THREAD

**Issue by arthurlutz** [bpetit/isso_ynh#2]
Following my test on #1 I see that the installer uses python2 https://posativ.org/isso/docs/install/ mentions support for python3. It might be a good idea to switch to python3 since python2 is end-of-life.

**Comment by bpetit** [comment]
You're absolutely right. I strongly consider to continue my work on buster and python3 exclusively.
